### PR TITLE
HC 967 update trademark articles and categories

### PR DESCRIPTION
**Pull request by Renat0Ribeir0**
**Changes proposed in this pull request**

_**1. Updated structure**_

(Category) About Arduino

---(Section) Compatible products
--------(Article) Developing compatible products
--------(Article) _Distributing products based on Arduino_ --> Software and Licensing based on Arduino

---(Section)  _Courses and Workshops_ AND _Publications and Websites_ --> Courses, workshops and publications
--------(Article) Guidelines for content creation

---(Section) How to spot a counterfeit
--------(Article) How to spot a counterfeit

--(Section) Donate to Arduino  ( _**Stays the same**_)

--(Section) My Arduino Account (_**Stays the same**_)

---(Section) Trademarks, logos and licenses
--------(Article) Arduino’s trademarks
--------(Article) Logo and Community Logo

_**2. New articles**_
- Created "Arduino's Trademarks" article
- Created "Arduino and Community" article
- Created "Developing Compatible Products article
- Created "Guidelines for content creation" article

_**3. Renamed articles**_
- Renamed "Distributing products based on Arduino" ->"Software and licensing based on Arduino"

_**4. Moved articles**_

-"How to spot a counterfeit"

 _**5. Removed articles**_

Community Section

Can-I-use-the-Arduino-Community-logo-on-or-in-connection-with-my-product-so-my-customers-easily-can-understand-what-it-is.md
Can-I-use-the-official-Arduino-logo-for-a-community-group
Can-I-use-the-official-Arduino-logo-on-my-blog-to-show-that-I-am-part-of-the-Arduino-Community
I-am-arranging-an-event-that-will-be-focused-on-Arduino-May-I-use-the-Arduino-logo-in-the-poster-and-on-other-promotional-materials
To-what-extent-am-I-allowed-to-customize-the-Community-logo
Why-cant-I-use-the-Community-logo-on-a-commercial-product-to-show-that-it-derived-from-the-Arduino-Community

Compatible Products Section

Can-I-design-my-own-Arduino-board-and-sell-it.md"
Can-I-further-develop-the-Arduino-IDE-and-make-my-own-distribution-of-it.md"
Can-I-include-Arduino-in-the-name-of-my-product-so-my-customers-easily-can-understand-what-it-is.md"
Can-I-include-the-Arduino-name-or-parts-of-it-in-my-company-name-since-my-business-is-similar-to-Arduino.md"
Can-I-make-a-variation-of-the-Arduino-logotype-and-use-it-on-my-product-since-it-is-based-on-Arduino-technology.md"
Can-I-use-the-Arduino-logo-on-or-in-connection-with-my-product-so-my-customers-easily-can-understand-what-it-is.md"
I-have-designed-a-product-that-has-an-Arduino-board-as-its-core-Do-I-have-to-make-my-product-open-source.md"
I-have-used-Arduino-for-my-project-do-I-need-to-release-my-source-code.md"
Is-it-allowed-to-copy-Arduino-boards.md"
I-want-to-make-my-own-boards-for-personal-use-or-for-use-by-my-company-only-Is-that-allowed.md"
Since-Arduino-uses-Creative-Commons-licenses-do-I-also-need-to-do-that-on-my-product.md"
What-makes-the-Arduino-board-unique.md"

Courses and Workshops

My-course-teaches-how-to-use-Arduino-technology-Can-I-use-the-Arduino-logo-in-the-certificate-released-upon-completing-my-course.md"
Can-I-use-photos-or-other-content-from-arduino-cc-in-my-course-material.md"
Can-I-use-screenshots-of-the-Arduino-IDE-software-in-my-course-material.md"
Can-I-use-the-Arduino-community-logo-for-my-course.md"
Can-I-use-the-Arduino-logo-in-course-material.md"
Can-I-use-the-Arduino-logo-to-advertise-my-course-or-workshop-if-it-uses-Arduino-technology.md"
Can-I-use-the-Arduino-logo-to-identify-my-course-if-it-uses-Arduino-technology.md"
Can-I-use-the-Arduino-logo-to-identify-non-commercial-content.md"
Do-I-need-to-include-a-trademark-acknowledgment-in-my-course-material.md"
How-can-I-brand-my-course-about-Arduino-technology-without-violating-the-Arduino-trademarks.md"
I-am-planning-to-develop-courses-using-Arduino-boards-do-I-need-special-permissions.md"
I-developed-a-course-that-uses-Arduino-technology-Can-I-commercialize-it-or-does-it-have-to-be-free.md"
If-I-develop-a-course-that-uses-Arduino-technology-do-all-the-components-of-the-kits-need-to-be-original-Arduino.md"
If-I-sell-my-course-that-uses-Arduino-technology-do-I-need-to-pay-Arduino-a-license-fee-or-a-royalty-fee.md"
If-I-use-Arduino-boards-or-the-Arduino-IDE-in-my-course-do-I-have-to-mention-it-in-the-name-or-description.md"

Publications and Websites

Can-I-use-the-Arduino-logo-in-my-publication-or-website.md"
Can-I-use-the-Arduino-logo-to-identify-my-publication-or-website.md"
Can-I-use-the-word-Arduino-in-the-title-of-my-publication-or-website.md"
How-can-I-give-acknowledgement-to-Arduino-for-using-the-trademarks-of-Arduino-SA-in-my-publication-website.md"
I-would-like-to-write-a-publication-or-start-a-website-about-Arduino-do-I-need-any-special-permission.md"
Can-I-use-my-own-photos-images-of-Arduino-products-in-my-publication-website.md"
Can-I-use-photos-or-other-content-from-arduino-cc-in-my-publication-website.md"
Can-I-use-screenshots-of-the-Arduino-IDE-software-in-my-publication-website.md"

Trademarks, Logos and Licenses
Can-I-get-permission-to-use-the-trademarks-of-Arduino-on-my-product-book-blog-etcetera.md"
Does-Arduino-have-any-other-trademarks-aside-from-the-word-and-infinity-eight-logo.md"
Do-I-need-to-include-the-registered-trademark-symbol-after-each-use-of-the-word-Arduino.md"
How-can-you-be-trademarked-when-you-are-open-source.md"
What-is-a-trademark.md"
Who-can-use-Arduino-trademarks.md"
Are-the-IDE-examples-free-to-use.md"

**Please check**
- [x] This pull request changes an article title
- [x] This pull request removes an article

**TO DO**
Fix links in "Arduino and Community" after publishing
Archive Deleted Articles

### CODE REVIEW

File: content/About Arduino/Compatible Products/Developing-Compatible-Products.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+If you want your compatible product to carry a logotype, you should develop your own. Your logo should be clearly distinguishable from the Arduino logo, even if your product is derived from the Arduino technology.
+
+You are not allowed to use the Arduino logo in connection with the compatible product you have developed. This includes: printing the logo on the product, on the packaging, on any promotional material advertising your product (including social media).
+
+Modified versions of the Arduino logo are considered trademark violations.
+
+## Name
+
+It is understandable that compatible brands want to make a reference to Arduino®, in order for their customers to easily identify what type of product it is.
```
Comment (seaxwi) [inline content/About Arduino/Compatible Products/Developing-Compatible-Products.md:42]:
```suggestion
It is understandable that compatible brands want to make a reference to Arduino<sup>®</sup>, in order for their customers to easily identify what type of product it is.
```

File: content/About Arduino/Compatible Products/Developing-Compatible-Products.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    <td><img src="img/ButterflyBoard_good_example.png" alt="Gray butterfly logo with "Butterfly BOARD" written on the side"></td>
+    <td><img src="img/ButterflyBoard_bad_example.png" alt="Arduino Infinity logo with "Butterfly BOARD" written on the side"></td>
+  </tr>
+  <tr>
+    <td>Product Category</td>
+    <td  style="vertical-align: top;">
+      Product menu:
+      <li>Arduino®<ul>
+        <li>Arduino® boards</li>
+        <li>Arduino® Kits</li>
```
Comment (seaxwi) [inline content/About Arduino/Compatible Products/Developing-Compatible-Products.md:94]:
```suggestion
      <li>Arduino<sup>®</sup><ul>
        <li>Arduino<sup>®</sup> boards</li>
        <li>Arduino<sup>®</sup> Kits</li>
```

File: content/About Arduino/Courses, Workshops and Publications/Guidelines-for-content-creation.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+All content on our website, including text and images, is distributed under Creative Commons license. This means you can use them on your own derived works, in part or completely, as long as you also adopt the same license.
+
+* See [Software and Licensing based on Arduino](https://support.arduino.cc/hc/en-us/articles/4415094490770-Distributing-products-based-on-Arduino).
+
+> If your content includes the use of components (for example, if you develop a video tutorial), you are free to use whichever brand you choose. In other words, you do not have to use original Arduino products. However, you must provide correct descriptions of the components you use. If you are using a compatible board, you may not call it “Arduino”.
+
+## Trademark Acknowledgments
+
+All uses of the trademarks of Arduino SA should include an acknowledgement (for example: “Arduino® is a trademark of Arduino SA”). Please note that you should also include the registered trademark symbol the first time Arduino is mentioned in text: ARDUINO® or Arduino®.
```
Comment (seaxwi) [inline content/About Arduino/Courses, Workshops and Publications/Guidelines-for-content-creation.md:55]:
```suggestion
All uses of the trademarks of Arduino SA should include an acknowledgment (for example: “Arduino<sup>®</sup> is a trademark of Arduino SA”). Please note that you should also include the registered trademark symbol the first time Arduino is mentioned in text: ARDUINO<sup>®</sup> or Arduino<sup>®</sup>.
```

File: content/About Arduino/Compatible Products/Developing-Compatible-Products.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+* “Compatible with Arduino”
+* “For Arduino”
+* “Based on Arduino”.
+
+> Always place the word Arduino last.
+
+### Catalogue and websites
+
+The word Arduino is a brand name, not a category. Therefore, you should clearly separate original Arduino products and compatible products in your catalogue or website.
```
Comment (seaxwi) [inline content/About Arduino/Compatible Products/Developing-Compatible-Products.md:54]:
```suggestion
### Catalog and websites

The word Arduino is a brand name, not a category. Therefore, you should clearly separate original Arduino products and compatible products in your catalog or website.
```

File: content/About Arduino/Courses, Workshops and Publications/Guidelines-for-content-creation.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+You are allowed to use screenshots of Arduino IDE in your course content, provided that you mention its source. You are also free to use images of Arduino products in your course or publication.
+
+All content on our website, including text and images, is distributed under Creative Commons license. This means you can use them on your own derived works, in part or completely, as long as you also adopt the same license.
+
+* See [Software and Licensing based on Arduino](https://support.arduino.cc/hc/en-us/articles/4415094490770-Distributing-products-based-on-Arduino).
+
+> If your content includes the use of components (for example, if you develop a video tutorial), you are free to use whichever brand you choose. In other words, you do not have to use original Arduino products. However, you must provide correct descriptions of the components you use. If you are using a compatible board, you may not call it “Arduino”.
+
+## Trademark Acknowledgments
```
Comment (seaxwi) [inline content/About Arduino/Courses, Workshops and Publications/Guidelines-for-content-creation.md:53]:
```suggestion
## Trademark acknowledgments
```

File: content/About Arduino/Courses, Workshops and Publications/Guidelines-for-content-creation.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+## Licenses and fees
+
+If you decide to open a website about Arduino, write a book, or develop a course or a workshop, you are free to do so, and you do not have to ask us for permission or pay any license fees. Similarly, if you commercialize your course or publication, you do not have to pay Arduino any royalty fees.
+
+## Course content
+
+You are allowed to use screenshots of Arduino IDE in your course content, provided that you mention its source. You are also free to use images of Arduino products in your course or publication.
+
+All content on our website, including text and images, is distributed under Creative Commons license. This means you can use them on your own derived works, in part or completely, as long as you also adopt the same license.
```
Comment (seaxwi) [inline content/About Arduino/Courses, Workshops and Publications/Guidelines-for-content-creation.md:47]:
```suggestion
All content on our website, including text and images, is distributed under a Creative Commons license. This means you can use them on your own derived works, in part or completely, as long as you also adopt the same license.
```
Also: which CC license?

File: content/About Arduino/Trademarks, Logos and Licenses/Arduino-Trademarks.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Arduino Trademarks"
+---
+
+A trademark is a word or symbol representing a company or product. It is used to identify a specific source of goods or service and protect that source from confusion with other sources. Trademarks are recognizable by the symbols ™ or ®.
```
Comment (seaxwi) [inline content/About Arduino/Trademarks, Logos and Licenses/Arduino-Trademarks.md:5]:
```suggestion
A trademark is a word or symbol representing a company or product. It is used to identify a specific source of goods or services and protect that source from confusion with other sources. Trademarks are recognizable by the symbols ™ or ®.
```

File: content/About Arduino/Trademarks, Logos and Licenses/Arduino-Trademarks.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+>![Arduino Pro logo with "TM" symbol highlighted](img/ArduinoTrademark_TM.jpg) ![infinity logo with "®" symbol highlighted](img/ArduinoTrademark_R.jpg)
+
+Most hardware designs of Arduino are open-source, but the company's name and logos are not. Copying and further developing Arduino's open-source products into their own thing is allowed, as long as the name Arduino, the infinity logotype and other trademarks are respected.
```
Comment (seaxwi) [inline content/About Arduino/Trademarks, Logos and Licenses/Arduino-Trademarks.md:9]:
```suggestion
Most hardware designs of Arduino are open-source, but the company's name and logos are not. Copying and further developing Arduino's open-source products into their own thing is allowed, as long as the name Arduino, the infinity logotype, and other trademarks are respected.
```

File: content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+
+ > The logo can sometimes be accompanied by the word mark *Arduino* or appear on its own.
+
+The Arduino logo is a registered trademark and cannot be used by third parties without the trademark owner’s authorization. Generally speaking, you **cannot** use the Arduino Logo (with or without the word *Arduino*) to identify or advertise your products and/or services. This includes:
+
+* Compatible products <!-- link to Compatible products section-->
+* Publications <!-- link to Courses, workshops and publications section-->
+* Online content.
+
+> Exception when the Arduino Logo is used in a descriptive or explanatory way. For example, the Arduino Logo may be used within a text explaining what Arduino is.
```
Comment (seaxwi) [inline content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md:23]: Grammar doesn't seem quite right

File: content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+
+Please be aware that even modified/customized versions of the Arduino Logo are considered trademark violations.
+
+* [See examples](#examples) of correct and incorrect uses of Arduino logos. Note that the examples are for illustrative purposes.
+
+<h2 id="community-logo">Community Logo</h2>
+
+The Arduino Community Logo ([download here](https://www.arduino.cc/en/Trademark/CommunityLogo)) has been developed by Arduino to give users the chance to identify non commercial pages, groups, events and initiatives connected to Arduino.
```
Comment (seaxwi) [inline content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md:31]:
```suggestion
The Arduino Community Logo ([download here](https://www.arduino.cc/en/Trademark/CommunityLogo)) has been developed by Arduino for users to identify non-commercial pages, groups, events, and initiatives related to Arduino.
```
For me, "connected" seems to imply some official endorsement, which I don't think is the case.

File: content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+
+> ![Infinity Arduino Community Logo](img/ArduinoCommunityLogo.png)
+
+The Arduino Community Logo can be used freely on, for example:
+
+* Social media community pages and groups, and to identify not for profit activities such as free events
```
Comment (seaxwi) [inline content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md:37]:
```suggestion
* Social media community pages and groups, and to identify not-for-profit activities such as free events
```
An event doesn't have to be free for it to be not-for-profit? Could be clarified.

File: content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+
+* Non commercial publications, courses and workshops. (e.g. when developing a not for profit video tutorial about Arduino, the Community Logo can be used in connection to it)
+
+The Arduino Community Logo **cannot** be used on:
+
+* Products or services that are sold commercially
+
+* Electronic boards (also non commercial)
```
Comment (seaxwi) [inline content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md:45]:
```suggestion
* Electronic boards (including non-commercial)
```

File: content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+
+**Note**: the Arduino Community Logo is shared with Attribution-NonCommercial-ShareAlike 3.0 Unported (CC BY-NC-SA 3.0) license. It is designed to allow personalization in the lower-right part in order to provide a free space on which to add the name of your group or organization. Apart from that, the community logo should not be altered, cropped, shortened or modified in any way. It is not allowed to incorporate another logo, trademark or commercial company name into the Community Logo.
```
Comment (seaxwi) [inline content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md:47]:
```suggestion
**Note**: the Arduino Community Logo is shared with a [Attribution-NonCommercial-ShareAlike 3.0 Unported (CC BY-NC-SA 3.0) license](https://creativecommons.org/licenses/by-nc-sa/3.0/). It is designed to allow personalization in the lower-right part in order to provide a free space on which to add the name of your group or organization. Apart from that, the community logo should not be altered, cropped, shortened, or modified in any way. It is not allowed to incorporate another logo, trademark, or commercial company name into the Community Logo.
```

**Review by seaxwi — CHANGES_REQUESTED** [review]
Nice work! Haven't done a thorough read yet, but here's what stood out to me.

* Can't recall if the section structure was locked in, but having a single article in a section with the same name does seem a bit strange to me. Can we motivate putting _Guidelines for content creation_ (with headings such as _Logo_ and _Licenses..._) in its own section instead of the "Trademarks, Logos and Licenses" section?
* Consider changing instances (including the section name) of "Courses, workshops and publications" to "Courses, workshops, and publications".
* See various comments and suggestions

File: content/About Arduino/Compatible Products/Software-and-licensing-based-on-Arduino.md [hunk 14]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Distributing products based on Arduino"
+title: "Software and licensing based on Arduino"
```
[the remark this answers is not in the document]
Comment (maria-tonini) [reply content/About Arduino/Compatible Products/Software-and-licensing-based-on-Arduino.md:2]: great suggestion, yes.

File: content/About Arduino/Compatible Products/Developing-Compatible-Products.md [hunk 15]
```diff
@@ -0,0 +1,118 @@
+---
+title: "Developing Compatible Products"
```
[the remark this answers is not in the document]
Comment (maria-tonini) [reply content/About Arduino/Compatible Products/Developing-Compatible-Products.md:2]: perfect

File: content/About Arduino/Compatible Products/Developing-Compatible-Products.md [hunk 16]
```diff
@@ -0,0 +1,118 @@
...
+
+* It is **not allowed** to print the word *Arduino* or the  Arduino logo on the board. Manufacturing and selling counterfeits is illegal.
+
+* It is **not allowed** to call your compatible product *Arduino*, even if you copied the design from Arduino. Your finished product is a product compatible with Arduino and should have its own name and brand.
+
+## Logo
+
+If you want your compatible product to carry a logotype, you should develop your own. Your logo should be clearly distinguishable from the Arduino logo, even if your product is derived from the Arduino technology.
+
+You are not allowed to use the Arduino logo in connection with the compatible product you have developed. This includes: printing the logo on the product, on the packaging, on any promotional material advertising your product (including social media).
```
[the remark this answers is not in the document]
Comment (maria-tonini) [reply content/About Arduino/Compatible Products/Developing-Compatible-Products.md:36]: good catch

File: content/About Arduino/Compatible Products/Developing-Compatible-Products.md (hunk 4 above)
Comment (maria-tonini) [reply content/About Arduino/Compatible Products/Developing-Compatible-Products.md:54]: yes good, sometimes I tend to use the British spelling so thank you for reviewing this (also the comment below on acknowledgments)

**Review by maria-tonini — APPROVED** [review]

File: content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md (hunk 9 above)
Comment (Renat0Ribeir0) [reply content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md]: See [Update grammar](https://github.com/arduino/help-center-content/pull/43/commits/4566d7f834ccdf5f8cc83dda5d59f581d3a90a45)

File: content/About Arduino/Courses, Workshops and Publications/Guidelines-for-content-creation.md (hunk 6 above)
Comment (Renat0Ribeir0) [reply content/About Arduino/Courses, Workshops and Publications/Guidelines-for-content-creation.md]: See [7a3100a](https://github.com/arduino/help-center-content/pull/43/commits/7a3100aba033099549130334d74237c6c939e73b)

File: content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md (hunk 11 above)
Comment (Renat0Ribeir0) [reply content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md]: See [3443244](https://github.com/arduino/help-center-content/pull/43/commits/34432441d514c8be223f7a746b121920c6356980) Maybe we don't need to give examples of non-for-profit activities.

File: content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md (hunk 9 above)
Comment (seaxwi) [reply content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md]: > do you mean line 23?
Yes

> See [4566d7f](https://github.com/arduino/help-center-content/pull/43/commits/4566d7f834ccdf5f8cc83dda5d59f581d3a90a45)
This works for me!

File: content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md (hunk 11 above)
Comment (seaxwi) [reply content/About Arduino/Trademarks, Logos and Licenses/Arduino-and-Community-logos.md]: @Renat0Ribeir0 Yeah, that makes sense.

File: content/About Arduino/Compatible Products/Developing-Compatible-Products.md (hunk 4 above)
Comment (seaxwi) [reply content/About Arduino/Compatible Products/Developing-Compatible-Products.md]: @maria-tonini Totally understand, I also find the British variants more natural in many cases

**Review by seaxwi — APPROVED** [review]

**Comment by seaxwi** [comment]
@maria-tonini Thanks for reviewing!

@Renat0Ribeir0 `About Arduino/Compatible Products/Developing-Compatible-Products.md` needs to be renamed to match the new tile. otherwise it LGTM  👍

**Pull request merged**